hls4ml/converters/onnx/merge.py

```diff
@@ -21,22 +33,10 @@ def parse_merge_layer(reader, node, inputs_map, input_shapes, graph, config):
         layer['op'] = layer['class_name'].lower() + f'{rank}d'
         layer['axis'] = get_onnx_attribute(node, 'axis')
 
-        # Calculate output shape
-        new_dim = sum(
-            [x.type.tensor_type.shape.dim[layer['axis']].dim_value for x in graph.value_info if x.name in node.input]
-        )
-        output_shape[layer['axis']] = new_dim
-
-    elif layer['class_name'] == 'Add':
```

Comment:
Just for my understanding, what is the reason that BiasAdd is no longer supported?

Reply:
It is handled as just a regular merge with add.

Reply:
Merge with constant becomes a batchnorm, and it can then be fused with a dense layer if there are no conflicts.

Reply:
I did wonder if adds and scales should be kept separate, but there's a whole infrastructure setup already for scale+add, which is what batch norms become in the end. (We don't support properly quantized batch-norms, where the mean, etc., is quantized, by the way, even for QKeras.)

Reply:
Thanks for the explanation. Sounds ok to me.